### DISCUSSION TITLE
Correct AT!ENTERCND argument for EM7345?

### DISCUSSION THREAD

**Discussion by RobertBuhren** [danielewood/sierra-wireless-modems#35] [General] [answered]
Hi,

I came here because I wanted to make use of my EM7345 LTE Modem. When hotplugging it during S3 sleep, the modem works fine. However, during cold boot, my laptop (Thinkpad X1) stops booting due to the network device whitelist. 

Unfortunately I can't use the boot enumeration trick as the AT!ENTERCND="A710" command only gives me an error. I suspect that the password is different for the EM7345. Any ideas how to get around this? Is the password for the EM7345 known?

Regards,

Robert

**Comment by danielewood** [comment, marked as answer]
If `AT!ENTERCND="A710"` is not working, that means the OEM customized it. There is nothing you can do that would be worth expending any effort to fix this. A replacement modem would be under $30, I suggest you take that route and get a better modem in the process.
